p4-16/spec/P4-16-spec.mdk

```diff
@@ -6584,9 +6584,10 @@ table is an abstract finite map that is given a key and produces as a
 result either an action or a "miss" indication, as described in
 Section [#sec-mau-semantics].
 
-If a table has no `key` property, then it contains no look-up
-table, just a default action---i.e., the associated lookup table is
-always the empty map.
+If a table has no `key` property, or if the value of its `key`
+property is the empty list, then it contains no look-up table, just a
```

Comment:
Would it be possible to add an example of "empty list" as in P4 this is not obvious?

Reply:
Good idea. Maybe just a parenthetical?